src/extract/rejection.rs

```diff
@@ -39,13 +39,6 @@ define_rejection! {
     pub struct HeadersAlreadyExtracted;
 }
 
-define_rejection! {
-    #[status = BAD_REQUEST]
-    #[body = "Query string was invalid or missing"]
-    /// Rejection type for [`Query`](super::Query).
-    pub struct QueryStringMissing;
-}
```

Comment:
We can't remove this as thats a breaking change. We can deprecate it

```rust
define_rejection! {
    #[status = BAD_REQUEST]
    #[body = "Query string was invalid or missing"]
    #[deprecated(since = "0.1.3", note = "No longer used. See https://github.com/tokio-rs/axum/pull/117")]
    /// Rejection type for [`Query`](super::Query).
    pub struct QueryStringMissing;
}
```

and then remove it in 0.2